scripts/compose/grafana/provisioning/dashboards/log-viewer.json

```diff
@@ -0,0 +1,200 @@
+{
```

Comment:
Could this be YAML as well for consistency?

Reply:
No, dashboards have to be in JSON, I have added a link to the docs to the commit message.